src/custom/hooks/useFetchProfile.ts

```diff
@@ -6,19 +6,26 @@ import { ProfileData } from 'api/gnosisProtocol/api'
 export default function useFetchProfile() {
   const { account, chainId } = useActiveWeb3React()
   const [profileData, setProfileData] = useState<ProfileData | null>(null)
+  const [error, setError] = useState('')
 
   useEffect(() => {
     async function fetchAndSetProfileData() {
-      if (chainId && account) {
+      if (!chainId || !account) {
```

Comment:
@ramirotw I have denied the condition to keep only one level and avoid nesting two conditions.